README.md

```diff
@@ -66,6 +66,16 @@ These are:
   Go to `Settings > Build, Execution, Deployment > Compiler` and change
   `Build process heap size (Mbytes)` to something large, eg 2048.
 
+* **Pulling docker containers from ECR**
+  
+  You'll need to log into ECR before local docker can pull from there:
+  ```bash
+  # bash etc
+  eval $(AWS_PROFILE=platform-dev aws ecr get-login --no-include-email)
```

Comment:
Depends on your setup but I think this should be `platform-developer`?

Reply:
That is the name of the role and is the name of the profile in the AWS config file I was given when I joined, at least

Reply:
```suggestion
  eval $(AWS_PROFILE=platform-developer aws ecr get-login --no-include-email)
```